packages/FlexGrid/Col/Col.jsx

```diff
@@ -5,42 +5,65 @@ import { Subscriber } from 'react-broadcast'
 import { Col as ReactFlexboxGridCol } from 'react-flexbox-grid'
 
 import safeRest from '../../../shared/utils/safeRest'
+import { deprecate } from '../../../shared/utils/warn'
 import styles from './Col.modules.scss'
 
-const removeProps = ({
-  xs,
-  sm,
-  md,
-  lg,
-  xl,
-  xsOffset,
-  smOffset,
-  mdOffset,
-  lgOffset,
-  xlOffset,
-  ...rest
-}) => safeRest(rest)
-
-const Col = ({ span, offset, children, ...rest }) => (
-  <Subscriber channel="flex-grid">
-    {gutter => (
-      <ReactFlexboxGridCol
-        {...removeProps(rest)}
-        xs={span || true}
-        xsOffset={offset}
-        className={gutter ? styles.padding : styles.gutterless}
-      >
-        {children}
-      </ReactFlexboxGridCol>
-    )}
-  </Subscriber>
-)
+const removeProps = ({ xsOffset, smOffset, mdOffset, lgOffset, xlOffset, ...rest }) =>
+  safeRest(rest)
+
+const Col = ({ span, offset, xs, sm, md, lg, xl, children, ...rest }) => {
+  if (span) {
+    deprecate(
+      'core-flex-grid',
+      'The span prop is deprecated due to the addition of the xs, sm, md, lg and xl responsive props. Please replace the span prop in your Col definition with the xs prop for identical functionality.'
+    )
+  }
+
+  return (
+    <Subscriber channel="flex-grid">
+      {gutter => (
+        <ReactFlexboxGridCol
+          {...removeProps(rest)}
+          xs={xs || span || true}
+          sm={sm || 0}
+          md={md || 0}
+          lg={lg || 0}
+          xl={xl || 0}
+          xsOffset={offset}
+          className={gutter ? styles.padding : styles.gutterless}
+        >
+          {children}
+        </ReactFlexboxGridCol>
+      )}
+    </Subscriber>
+  )
+}
 
 Col.propTypes = {
   /**
-   * Span the specified number of columns.
+   * @deprecated Span the specified number of columns.
    */
   span: PropTypes.oneOf([1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12]),
+  /**
+   * Specify number of columns within the 'xs' breakpoint range. (0px - 575px)
+   */
+  xs: PropTypes.oneOf([1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12]),
+  /**
+   * Specify number of columns within the 'sm' breakpoint range. (576px - 767px)
+   */
+  sm: PropTypes.oneOf([1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12]),
+  /**
+   * Specify number of columns within the 'md' breakpoint range. (768px - 991px)
```

Comment:
I suggest to remove the pixel values from these comments. Not sure its helpful, and we already have the breakpoints listed elsewhere in the docs. Maybe the FlexGrid docs can just link to where the table is.

Reply:
Sure, although maybe then that table should be made to sound more general-purpose rather than just intended for the responsive component in the docs.

Reply:
Agreed. That table represents the design system breakpoints and every component that uses breakpoints should all be using the same values.